php/libraries/NDB_Page.class.inc

```diff
@@ -569,6 +569,7 @@ class NDB_Page
                   $baseurl . '/js/polyfiller.js',
                   $baseurl . '/bootstrap/js/bootstrap.min.js',
                   $baseurl . '/js/components/react.breadcrumb.js',
+                  $baseurl . '/js/components/Form.js',
```

Comment:
NDB_BVL_Instrument will potentially need to use react form elements in the future and it extends NDB_Page.